docs(operator): create a doc for installing Spinnaker using Openshift

cristhian-castaneda opened the pull request:
…in a Openshift cluster.

cristhian-castaneda committed:
docs(operator): create a doc for installing Spinnaker using Operator in a Openshift cluster.

dogonthehorizon commented on content/en/docs/installation/guide/redhat-marketplace.md:
```diff
@@ -0,0 +1,309 @@
+2. Browse to the [Red Hat Marketplace](https://marketplace.redhat.com/en-us?_ga=2.94000664.441560370.1604955054-123808014.1597373423) and log in or create a new account.
+3. Follow the [instructions](https://marketplace.redhat.com/en-us/documentation/clusters?&_ga=2.94000664.441560370.1604955054-123808014.1597373423#register-openshift-cluster-with-red-hat-marketplace) to register your OpenShift cluster with the Red Hat Marketplace.
+4. When prompted `Would you like to go back to the Red Hat Marketplace now? [Y/n]`, type `Y` and the Red Hat Marketplace page is opened in your browser.
+5. Click <b>My software</b> >  <b>Visit the Marketplace.</b>
```
I think `<strong>` is preferred these days over `<b>`; are we getting these instructions straight from RedHat?

aimeeu replied:
I'll be replacing the HTML with Markdown. :)

cristhian-castaneda replied:
Thank you @aimeeu

cristhian-castaneda replied:
@dogonthehorizon kind of, these instructions are implicit here https://marketplace.redhat.com/en-us/documentation/getting-started

dogonthehorizon commented on content/en/docs/installation/guide/redhat-marketplace.md:
```diff
@@ -0,0 +1,309 @@
+
+This document covers the following high-level steps:
+
+1. Configure application and pipeline configuration storage 
```
```suggestion
1. Configuring application and pipeline storage 
```

So that you maintain the tenses in 2. and 3.

Also random note: in markdown you can create numbered lists with `1.` by itself, that way you don't have to renumber elements if you add/remove items from the list.

Example:
```markdown
1. this is
1. a numbered
1. list!
```

Rendered:

1. this is
1. a numbered
1. list!

cristhian-castaneda replied:
Thanks for the note @dogonthehorizon !

Commit:
Update content/en/docs/installation/guide/redhat-marketplace.md

Co-authored-by: Fernando Freire <fernando.freire@armory.io>

aimeeu committed:
Edits and additions

- rename file
- move images to images/installation/guide/redhat
- remove section on how to create s3 bucket
- add URL links to storage types
- add links to RH docs
- add more screen shots
- changed some content
- added content

aimeeu commented:
Thanks @cristhian-castaneda for the content!

I did a few things:
- renamed file, added "install-" to the beginning
- moved `redhat` images folder deeper,  to images/installation/guide/redhat
- remove section on how to create s3 bucket
- added URL links to storage types
- added links to RH docs
- added links to our Operator config pages
- removed Minio and MySQL from the persistent storage types list because they aren't supported according to https://docs.armory.io/docs/installation/operator-reference/persistent-storage/.    If they are, then we need to update the persistent storage config reference and this PR.
- added more screen shots to make is easier for readers to locate the screens we talk about
- rearranged some content
- added content to areas where I was confused because I don't know OpenShift as well as you do :)

aimeeu commented:
Deploy preview:   https://deploy-preview-302--armory-docs.netlify.app/docs/installation/guide/install-redhat-marketplace/

aimeeu committed:
fix typo
